mne/io/brainvision/brainvision.py

```diff
@@ -308,18 +346,41 @@ def _get_vhdr_info(vhdr_fname, eog, misc, response_trig_shift, scale):
     cals = np.empty(info['nchan'])
     ranges = np.empty(info['nchan'])
     cals.fill(np.nan)
+    ch_dict = list()
```

Comment:
Why make a list then pass to dict constructor later, instead of initializing a dict and setting key values within the loop as normal?